### PR TITLE
[DS-3795] Manage versions of some buggy transitive dependencies.

### PR DESCRIPTION
**Pull request by mwoodiupui**
https://jira.duraspace.org/browse/DS-3795
Ported #2032 from master.

### CODE REVIEW

**Comment by mwoodiupui** [comment]
This may actually be better than the original.  I made more use of dependencyManagement rather than small tweaks to the puzzling exclude-and-depend method.

**Comment by tdonohue** [comment]
@mwoodiupui : could you link in the PRs that this is a port from?  The DS-3795 ticket is becoming a bit unwieldy.  Is this a port of #2032 alone, or are there other changes included here?

Also, obviously the goal would be to sync `6.x` and `master`.  As this seems to be a new implementation (per your last comment), we'd need to resync the `master` branch with these changes again.

File: pom.xml [hunk 1]
```diff
@@ -1429,7 +1428,7 @@
             <dependency>
                 <groupId>com.fasterxml.jackson.core</groupId>
                 <artifactId>jackson-databind</artifactId>
-                <version>${jackson.version}</version>
+                <version>2.8.11.1</version> <!-- TODO sync. with jackson.version -->
```
Comment (terrywbrady) [inline pom.xml]: Why isn't the jackson.version variable in use here?  Could more explanation be added for the choice of this version?

Comment (mwoodiupui) [reply pom.xml]: Because jackson-databind-2.8.11 still has the bug.  There is no 2.8.11.1 of the other Jackson components.

**Review by terrywbrady — APPROVED** [review]
+1 Tested legacy rest and rest reports.

**Review by kshepherd — APPROVED** [review]
+1, tested XMLUI and XOAI with updated dependencies and all is working well, merging now, thanks @mwoodiupui

**Comment by tdonohue** [comment]
Added the `port to master` flag as this implementation is slightly improved from the `master` version.  See @mwoodiupui's comment above:
> This may actually be better than the original. I made more use of dependencyManagement rather than small tweaks to the puzzling exclude-and-depend method.

We need to port this minor improvement back to `master`.

**Pull request merged**